applications/alert-stream-broker/charts/alert-stream-schema-registry/templates/sync-schema-job.yaml

```diff
@@ -18,12 +18,16 @@ spec:
         app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
         app.kubernetes.io/instance: {{ .Release.Name | quote }}
         helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+      annotations:
+        description: "Load schemas into registry."
     spec:
       restartPolicy: Never
       containers:
       - name: sync-schema-job
-        image: "{{ .Values.schemaSync.image.repository }}:{{ .Values.schemaSync.image.tag | default .Chart.AppVersion }}"
+        #image: "{{ .Values.schemaSync.image.repository }}:{{ .Values.schemaSync.image.tag | default .Chart.AppVersion }}"
```

Comment:
do we still need the commented line?

Reply:
I can probably remove it. I was switching back and forth initially, but using the digest is more fine grained and makes sure that the correct docker image gets used, so its probably better to stick with that.